### ISSUE TITLE
Unable to start after upgrading to RC11.1 from RC10

### ISSUE THREAD

**Issue by RobotHanzo** [CloudNetService/CloudNet-v3#1544]
### Stacktrace

```
Error opening zip file or JAR manifest missing : /home/ubuntu/server/temp/caches/wrapper.jar
```

### Actions to reproduce

Start the server

### CloudNet version

.11 02:55:21.610] INFO :
[06.11 02:55:21.610] INFO : CloudNet Blizzard 4.0.0-RC12-SNAPSHOT 7aaa1cba
[06.11 02:55:21.610] INFO : Discord: <https://discord.cloudnetservice.eu/>
[06.11 02:55:21.611] INFO :
[06.11 02:55:21.611] INFO : ClusterId: c6b7bcad-****-401c-****-069f7903704c
[06.11 02:55:21.611] INFO : NodeId: Node-1
[06.11 02:55:21.611] INFO : Head-NodeId: Node-1
[06.11 02:55:21.611] INFO : CPU usage: (P/S) 3.18/18.86/100%
[06.11 02:55:21.611] INFO : Node services memory allocation (U/R/M): 8448/12544/60000 MB
[06.11 02:55:21.611] INFO : Threads: 61
[06.11 02:55:21.612] INFO : Heap usage: 86/256MB
[06.11 02:55:21.612] INFO : JVM: Eclipse Adoptium 23 (OpenJDK 64-Bit Server VM 23.0.1+11)
[06.11 02:55:21.612] INFO : Update Repo: CloudNetService/launchermeta, Update Branch: nightly

### Other

I tried using both the beta and nightly branch, though none of which worked

### Issue uniqueness

- [X] Yes, this issue is unique. There are no similar issues.

**Comment by 0utplay** [comment]
Does this happen with a clean restart and deleting the temp folder before starting?

**Comment by RobotHanzo** [comment]
> Does this happen with a clean restart and deleting the temp folder before starting?

Yes, I tried deleting temp and redownloading the core jars, unfortunately neither worked

**Comment by RobotHanzo** [comment]
> Does this happen with a clean restart and deleting the temp folder before starting?

FYI, the full error is as below (found in the log file)
```
Error occurred during initialization of VM
agent library failed Agent_OnLoad: instrument
Error opening zip file or JAR manifest missing : /home/ubuntu/server/temp/caches/wrapper.jar
```
Also, I use the docker module with the image `eclipse-temurin:23-jdk-noble`

**Comment by 0utplay** [comment]
Does it also happen without docker? Can you provide an minimal reproducer?

**Comment by RobotHanzo** [comment]
> Does it also happen without docker? Can you provide an minimal reproducer?

I tried creating a completely new  instance both with and without docker, interestingly they both worked
I cant figure out whats causing this (I tried disabling all my custom modules, it didn’t help)

**Comment by RobotHanzo** [comment]
> Does it also happen without docker? Can you provide an minimal reproducer?

However, disabling docker on the broken instance fixed the issue

**Comment by RobotHanzo** [comment]
Something ever weirder:
Copying the entire instance folder to somewhere else like `/tmp` fixed the issue
But as long as the instance folder is in the original parent directory (i.e. `/home/ubuntu/`), it wouldn't start even if renamed
Note: I made sure `wrapper.jar` can be read by the user running cloudnet

**Comment by RobotHanzo** [comment]
> Something ever weirder: Copying the entire instance folder to somewhere else like `/tmp` fixed the issue But as long as the instance folder is in the original parent directory (i.e. `/home/ubuntu/`), it wouldn't start even if renamed Note: I made sure `wrapper.jar` can be read by the user running cloudnet

A note for other devs seeking a temporary solution, if your compiles but can't pass tests, you can try the following to exclude cloudnet bridge's tainted jars from the test classpath:
```kotlin
tasks {
    test {
        classpath = files(classpath.filter { !it.name.contains("bridge-4.0.0-RC11.2") })

        useJUnitPlatform()
    }
}
```
